lib/image_compare/modes/base.rb

```diff
@@ -94,9 +94,9 @@ def update_bounds(x, y)
       end
 
       def area
-        area = include_rect.area
-        return area if exclude_rect.nil?
-        area - exclude_rect.area
+        total_area = @include_rect.area
+        total_exclude_area = @exclude_rects.nil? ? 0 : @exclude_rects.sum { |rect| rect.area }
```

Comment:
Please consider changing to `total_exclude_area = @exclude_rects&.sum(&:area) || 0`